Kernel/Arch/aarch64/RPi/InterruptController.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2022, Timon Kruiper <timonkruiper@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Format.h>
+#include <Kernel/Arch/aarch64/RPi/InterruptController.h>
+#include <Kernel/Arch/aarch64/RPi/MMIO.h>
+#include <Kernel/Interrupts/GenericInterruptHandler.h>
+
+namespace Kernel::RPi {
+
+// "7.5 Interrupts Registers"
```

Comment:
This reference to section 7.5 could use a document link. Is this a section number for the BCM 2387 SOC manual?

Reply:
Yep, added a link!